docs/built-in-models/training.md

```diff
@@ -0,0 +1,47 @@
+# Training on Encrypted Data
+
+Concrete ML offers the possibility to train [SGD Logistic Regression](../developer-guide/api/concrete.ml.sklearn.linear_model.md#class-sgdclassifier) on encrypted data. The [logistic regression training](../advanced_examples/LogisticRegressionTraining.ipynb) example shows this feature in action.
+
+This example shows how to instantiate a logistic regression model that trains on encrypted data:
+
+```python
+from concrete.ml.sklearn import SGDClassifier
+parameters_range = (-1.0, 1.0)
```

Comment:
maybe we should also explain a bit what `parameters_range` refers to and why it's set to -1, 1 here